shelley/chain-and-ledger/formal-spec/protocol-parameters.tex

```diff
@@ -7,15 +7,25 @@ \section{Protocol Parameters}
 and $\mathsf{T_{ppm}}$ represents the dependent type of a given protocol parameter.
 The parameters are listed in Figure~\ref{fig:defs:protocol-parameters},
 the first ten of which below are common to the Byron era.
+(NSF: I'm not sure the first ten are common to Byron. If they are, it's not
```

Comment:
I am not sure about this. @JaredCorduan ?

Reply:
it's looks like it has indeed gone stale, thank you @nfrisby . I'm looking at the Byron spec right now, and it seem that the ones still there that Shelley uses are: `a`, `b`, `maxBlockSize`, `maxTxSize`, `maxHeaderSize`.